README.md

```diff
@@ -8,6 +8,7 @@ It is officially maintained by the CoreOS team and meant to be a set of introduc
 
 When designing these guides and tools, the following considerations are made:
 
+* You must use a CoreOS version 773.1.0+ for the `kubelet` to be present in the image
```

Comment:
Can we drop this? This section was to call out the the motivations of the guides, not necessarily prerequisites. We've added the version requirement to the actual deploy docs.